### ISSUE TITLE
Scroll bar in FireFox when using fixedColumnsLeft

### ISSUE THREAD

**Issue by ghost** [handsontable/handsontable#10370]
### Description
When using FixedColumnsLeft setting, scrollbar which overlaps the last row in FireFox hides behind the fixed column.

I have used below CSS to add padding-bottom so that scrollbar is visible. but It does not get selected in firefox if it is on FixedColumns side I guess thats because of z-index of fixedColumns...

    @supports (-moz-appearance: none) {
           div.ht_master.handsontable.innerBorderLeft > div.wtHolder {
                padding-bottom: 15px !important;
          }
      }

### Steps to reproduce
1. open link in firefox and scroll 

### Demo
Here is demo link [Sandbox](https://codesandbox.io/s/strange-kare-gpuubq?file=/src/App.tsx)
https://codesandbox.io/s/strange-kare-gpuubq?file=/src/App.tsx

### Your environment
* React wrapper version: 11.1.0
* Handsontable version: 11.1.0

**Comment by adrianszymanski89** [comment]
Hi @jamshaid-tariq-geekybugs 

I checked the issue on Firefox, and I can't get the result you are describing:

![image](https://github.com/handsontable/handsontable/assets/92314130/9dd313c9-e53a-4f6f-bc7a-3e21df867cd2)

Do you use any kind of system or browser scaling? Any additional plugins in Firefox?

**Comment by jamshaid-tariq-geekybugs** [comment]
This is the Sandbox screenshot opened in firefox... you can see the scroll bar is overlapping last row of grid and also if you scroll all the way to left, It will hide under FixedColumns ie. first 6 columns of grid.
I have Firefox 113.0.1 (64-bit) and Ubuntu 22.04.2 LTS.  I think it should not be version dependent.

![Screenshot from 2023-05-15 19-00-48](https://github.com/handsontable/handsontable/assets/110381868/c41473b3-188c-4a3a-9332-f0c202d5e95d)

**Comment by adrianszymanski89** [comment]
@jamshaid-tariq-geekybugs 

Thank you for the screenshot. I'll test on other systems and see if I'll be able to replicate the issue.

**Comment by jamshaid-tariq-geekybugs** [comment]
any updates?

**Comment by adrianszymanski89** [comment]
Hi @jamshaid-tariq-geekybugs 

We tested it on other devices and still can't replicate the issue. Can you please tell me if you are using any scaling with Firefox?

**Comment by jamshaid-tariq-geekybugs** [comment]
I am not using any extension or plugin.
I have tested same app on another system even on ubuntu v22. but same result... I have tested app on Mac Firefox it works fine there..

**Comment by jamshaid-tariq-geekybugs** [comment]
@adrianszymanski89 any update?

**Comment by jamshaid-tariq-geekybugs** [comment]
are you able to produce the issue I have reported?

**Comment by adrianszymanski89** [comment]
Yes, we did reproduce it, but only with Firefox on Ubuntu. In all other configs, it worked fine.

**Comment by jamshaid-tariq-geekybugs** [comment]
> Yes, we did reproduce it, but only with Firefox on Ubuntu. In all other configs, it worked fine.

I used this CSS to add padding at bottom so that scrollbar shows up.

    @supports (-moz-appearance: none) {
         div.ht_master.handsontable.innerBorderLeft > div.wtHolder {
              padding-bottom: 15px !important;
         }
      }

 but with this setting I am unable to grab scrollbar with mouse when it is on the side of fixed columns. in my case, if scroll bar is at left side (i.e below first 7 columns in my case..). I think because of z-index of `ht_clone_left handsontable` table.

**Comment by GabrielHangor** [comment]
I have a similar issue. Row which is fixed at the bottom is overflowing the vertical scrollbar when the table is filled with particular amount of rows.

**Comment by vian94** [comment]
Hi @adrianszymanski89 
i have the same issue. reproduce on win11 firefox. any update?

**Comment by adrianszymanski89** [comment]
Hi @vian94 

Not yet. We checked if the most recent version of Handsontable has this problem, and it's still present. We will update this issue as soon as the bug is fixed.

**Comment by budnix** [comment]
Connected with https://github.com/handsontable/handsontable/issues/9305

**Comment by budnix** [comment]
Good news — this issue will be fixed soon. It's already in a [pull request](https://github.com/handsontable/handsontable/pull/13234). Once it's reviewed and merged, it will be released in upcoming versions.
